packages/mesos/extra/dcos-mesos-master.service

```diff
@@ -8,6 +8,7 @@ LimitNOFILE=infinity
 PermissionsStartOnly=True
 EnvironmentFile=/opt/mesosphere/environment
 EnvironmentFile=/opt/mesosphere/etc/mesos-master
+EnvironmentFile=-/opt/mesosphere/etc/mesos-master-extras
```

Comment:
I'd rather have this just before the `/run/dcos` one. That way the downstream overrides get the last say.